src/gobprepare/requests.py

```diff
@@ -0,0 +1,15 @@
+import requests
```

Comment:
All these files (requests, sql, csv, graphql, graphql_stream, etc) are put in the root of gobprepare. I'd put everything in a different module to keep the root of the gobprepare module clean and understandable. (If I were a new developer on the project, I should be directed to the code that is the core of the repo, the prepare_client. Auxiliary code such as request, sql, csv handling aren't important and should be put in their own namespace, or grouped in an api_importer module as long as api_importer is the only module using that code.)